examples/nlp/intent_slot_classification/data/mturk_utils.py

```diff
@@ -0,0 +1,197 @@
+# =============================================================================
```

Comment:
Why do we need this file? If we are not using this, lets not import it until it is needed.